packages/experiments/src/components/Text/Text.styles.ts

```diff
@@ -23,12 +28,12 @@ export const TextStyles = (props: IThemedProps<ITextProps>): ITextStyles => {
         fontWeight: typography.weights[weight]
       },
       color && {
-        color: semanticColors[color as keyof ISemanticColors] || palette[color as keyof IPalette] || color
+        color: semanticColors[color as keyof ISemanticTextColors]
```

Comment:
What if I just want to have the `Text` color be the default `bodyText` color? This seems frictiony to have to specify a `color` prop of `bodyText` just to get default coloring applied...

Reply:
Oh, I see now, you are setting `color` above on line 18. So if I change `bodyText` semantic slot, will the default text color change when falling back to the `default` variant?